account_financial_report_webkit/report/common_partner_balance_reports.py

```diff
@@ -80,6 +83,13 @@ def _get_account_partners_details(self, account_by_ids, main_filter,
                     get('init_balance', 0.0) + \
                     details[partner_id].get('debit', 0.0) - \
                     details[partner_id].get('credit', 0.0)
+
+            if display_partner == 'non-zero_balance':
+                details = {
+                    k: v
+                    for k, v in details.iteritems()
+                    if not float_is_zero(v['balance'], precision_digits=5)
```

Comment:
In my mind this is not meant to be the business concept of rounding.

Instead I really want to compare with zero, so that is just a technical workaround for handling floats, and 5 digits is an arbitrarily high precision.